File: python/code/130_pattern22.py
n = int(input("Enter n :- "))
for i in range(n):
    for j in range(i):
        print(i-j+1, end='')
    for k in range(i+1):
        print(k+1, end='')
    print()
    
# Output :-
# Enter n :- 5
# 1
# 212
# 32123
# 4321234
# 543212345  

File: c/code/130_pattern_22.c
#include<stdio.h>
int main() {
    int n;
    printf("Enter n :- ");
    scanf("%d",&n);
    for(int i=0;i<n;i++) {
        for(int j=0;j<i;j++) {
            printf("%d", i-j+1);
        }
        for(int k=0;k<=i;k++) {
            printf("%d", k+1);
        }
        printf("\n");
    }
}

/*

Output :-
Enter n :- 5
1
212
32123
4321234
543212345

*/